idflakies-core/src/main/java/edu/illinois/cs/dt/tools/detection/DetectorUtil.java

```diff
@@ -18,7 +21,8 @@
 import java.util.stream.Stream;
 
 public class DetectorUtil {
-    public static TestRunResult originalResults(final List<String> originalOrder, final Runner runner) {
+
+    public static TestRunResult originalResults(final File baseDir, final List<String> originalOrder, final Runner runner) {
```

Comment:
I think this can now be simplified to not have to pass in baseDir. Can then also remove the File import.